modular/gater/request_context.go

```diff
@@ -237,25 +263,26 @@ func (r *RequestContext) verifyTaskSignature(taskMsgBytes []byte, taskSignature
 	return addr, nil
 }
 
-// verifyOffChainSignature used to verify off-chain-auth signature, return (address, nil) if check succeed
-func (r *RequestContext) verifyOffChainSignature(requestSignature string) (sdk.AccAddress, error) {
+// verifySignatureForGNFD1Eddsa used to verify off-chain-auth signature, return (address, nil) if check succeed
+func (r *RequestContext) verifySignatureForGNFD1Eddsa(requestSignature string) (sdk.AccAddress, error) {
 	var (
-		signedMsg *string
-		err       error
+		err error
 	)
-	signedMsg, sigString, err := parseSignedMsgAndSigFromRequest(requestSignature)
+	_, sigString, err := parseSignedMsgAndSigFromRequest(requestSignature)
```

Comment:
Eddsa and ECDsa have same authorization, but why they are using the different functions to parse from header?